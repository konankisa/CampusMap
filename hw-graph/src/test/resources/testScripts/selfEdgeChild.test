# Creates an empty graph, adds a node to it,
# adds an edge to itself, and gets the child

# Create a graph
CreateGraph graph

# Add a node to graph
AddNode graph node

# Add an edge from node to node in graph
AddEdge graph node node edge

# Get the children of node
ListChildren graph node