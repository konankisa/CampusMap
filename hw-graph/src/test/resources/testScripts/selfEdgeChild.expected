# Creates an empty graph, adds a node to it,
# adds an edge to itself, and gets the child

# Create a graph
created graph graph

# Add a node to graph
added node node to graph

# Add an edge from node to node in graph
added edge edge from node to node in graph

# Get the children of node
the children of node in graph are: node(edge)